common/src/main/scala/ClassOneResult.scala

```diff
@@ -172,5 +172,52 @@ case class ClassOneResult(
 
   def grossPay: Explained[BigDecimal] = rowsInput.map{_.money}.sum gives
     s"grossPay: ${rowsInput.map(_.money).mkString(" + ")}"
-  
+
+
+  def employerPaid: Explained[BigDecimal] =
```

Comment:
is there a reason why most of the things in this case class are defs and not vals or lazy vals?

Reply:
I think the writers are lazy anyway, plus they're all placed into vals in the frontend facade so I don't think it shouldn't make any difference. 
If you prefer though I can change them to vals.

Reply:
it's not the end of the world really - just wondered if there was a special reason